### ISSUE TITLE
Doesn't make sense speeds of training RTX 3090 vs RTX 4090

### ISSUE THREAD

**Issue by FurkanGozukara** [kohya-ss/sd-scripts#827]
I have RTX 3090 TI, 64 GB RAM and 13900K CPU on Windows 10

On My PC 
LoRA gets 1.12 s/it with xformers enabled and gradient checkpoint disabled

Link > https://twitter.com/GozukaraFurkan/status/1704447704783261716

Command

`accelerate launch --num_cpu_threads_per_process=2 "./sdxl_train_network.py" --pretrained_model_name_or_path="F:/0 models/sd_xl_base_1.0.safetensors" --train_data_dir="E:/SD web ui with controlnet/stable-diffusion-webui/models/Stable-diffusion\img" --reg_data_dir="E:/SD web ui with controlnet/stable-diffusion-webui/models/Stable-diffusion\reg" --resolution="1024,1024" --output_dir="E:/SD web ui with controlnet/stable-diffusion-webui/models/Lora/model" --logging_dir="E:/SD web ui with controlnet/stable-diffusion-webui/models/Lora/log" --network_alpha="1" --save_model_as=safetensors --network_module=networks.lora --text_encoder_lr=0.0004 --unet_lr=0.0004 --network_dim=32 --output_name="Rank32Lora" --lr_scheduler_num_cycles="8" --no_half_vae --full_bf16 --learning_rate="0.0004" --lr_scheduler="constant" --train_batch_size="1" --max_train_steps="8320" --save_every_n_epochs="1" --mixed_precision="bf16" --save_precision="bf16" --cache_latents --cache_latents_to_disk --optimizer_type="Adafactor" --optimizer_args scale_parameter=False relative_step=False warmup_init=False --max_data_loader_n_workers="0" --bucket_reso_steps=64 --xformers --bucket_no_upscale --noise_offset=0.0`

On My PC
DreamBooth gets 1.5 s/it
xFormers disabled and Gradient checkpoint enabled

Link > https://twitter.com/GozukaraFurkan/status/1704442703067369557

Command

`accelerate launch --num_cpu_threads_per_process=4 "./sdxl_train.py" --pretrained_model_name_or_path="F:/0 models/sd_xl_base_1.0.safetensors" --train_data_dir="E:/SD web ui with controlnet/stable-diffusion-webui/models/Stable-diffusion\img" --reg_data_dir="E:/SD web ui with controlnet/stable-diffusion-webui/models/Stable-diffusion\reg" --resolution="1024,1024" --output_dir="E:/SD web ui with controlnet/stable-diffusion-webui/models/Stable-diffusion\model" --logging_dir="E:/SD web ui with controlnet/stable-diffusion-webui/models/Stable-diffusion\log" --save_model_as=safetensors --full_bf16 --output_name="SDXL_DB_Tuto" --lr_scheduler_num_cycles="8" --max_data_loader_n_workers="0" --learning_rate="1e-05" --lr_scheduler="constant" --train_batch_size="1" --max_train_steps="8320" --save_every_n_epochs="1" --mixed_precision="bf16" --save_precision="bf16" --cache_latents --cache_latents_to_disk --optimizer_type="Adafactor" --optimizer_args scale_parameter=False relative_step=False warmup_init=False weight_decay=0.01 --max_data_loader_n_workers="0" --bucket_reso_steps=64 --xformers --bucket_no_upscale --noise_offset=0.0`

Now with above exactly same commands here the results of RunPod - unix

Link > https://twitter.com/GozukaraFurkan/status/1704457802997969119

RTX 4090 dreambooth : 1.52 it/s
RTX 3090 dreambooth : 1s/it

Link > https://twitter.com/GozukaraFurkan/status/1704460038000034055

RTX 4090 LoRA : 1.02 s/it
RTX 3090 LoRA : 1.29 s/it
